Extension/package-lock.json

```diff
@@ -113,7 +113,7 @@
     "arr-flatten": {
       "version": "1.1.0",
       "resolved": "https://registry.npmjs.org/arr-flatten/-/arr-flatten-1.1.0.tgz",
-      "integrity": "sha512-L3hKV5R/p5o81R7O02IGnwpDmkp6E982XhtbuwSe3O4qOtMMMtodicASA1Cny2U+aCXcNpml+m4dPsvsJ3jatg==",
```

Comment:
i'm pretty sure you're not supposed to make changes to this file.

Reply:
@greazer You may need to update your npm since it is doing sha1 hashes. ```npm i -g npm```